code/obj/item/mops_cleaners.dm

```diff
@@ -592,6 +592,9 @@ TRASH BAG
 	if (!src.reagents)
 		return ..()
 
+	if (istype(target, /obj/ability_button))
```

Comment:
not a correct fix, this won't work for other ui elements

Reply:
@ZeWaka the bug doesn't happen on other UI elements, tested it on everything in the bottom bar, health/temp/stamina, and proper ability buttons. This is a special case because /obj/ability_button is what it is and not /atom/moveable/screen

Reply:
why isn't it

Reply:
just never got moved over, I actually found a comment in the code from you saying it should be moved over but it was out of your depth :P I'd like to try refactoring it in the future but in the mean time this was a more immediate fix

Reply:
i think it should just be refactored, this is a bit of a bandaid on a festering wound

Reply:
o7